js/activity.js

```diff
@@ -4178,7 +4180,8 @@ function Activity() {
                     errorMsg(_('project undefined'));
                     return;
                 }
-
+                textMsg(_(this.getCurrentProjectName()));
```

Comment:
I don't think we need to translate the project name.

Reply:
Now check @walterbender . I have done the changes.